addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -693,6 +725,53 @@ var SnippetEditor = Widget.extend({
             $optionsSection.toggleClass('d-none', options.length === 0);
         });
     },
+    /**
+     * Manages the overlay visibility towards animations/transitions
+     * states. This prevents having the overlay lagging behind the snippet
+     * css animation (with animation previews from 'website_animate' among others).
+     *
+     * @private
+     * @param {number} [animationsCountDelta=0] The delta to apply on the
+     * animations count as integer.
+     * @param {boolean} [animationTimeoutCall=false] is the call to the function made from
+     * the animation timeout function?
+     */
+    _manageOverlayMoves: function (animationsCountDelta = 0) {
+        const maxDuration = this._getTargetMaxAnimationAndTransitionDuration();
+        this._overlayMovesCounters += animationsCountDelta;
+        this._overlayMovesCounters = Math.max(0, this._overlayMovesCounters);
+        clearTimeout(this._postAnimationHandler);
+        clearTimeout(this._animationTimeoutHandler);
+        if (this._overlayMovesCounters > 0) {
+            this._toggleAllOverlaysVisibility(false);
```

Comment:
Is it possible to hide only the animated snippet ?

Reply:
As written in above comment, I think it would be overkill and it seems good to me that no overlays should be displayed during any snippet animation.